regulations/static/regulations/js/source/app-init.js

```diff
@@ -25,6 +26,7 @@ Backbone.$ = $;
     init: function() {
         Router.start();
         this.bindEvents();
+        $('table').stickyTableHeaders();
```

Comment:
As a general note - I don't think the table js code belongs in the -site repository.  Similar to css - I we need a way to introduce non-base JS in sites that extend from -site - thoughts?